.gitignore

```diff
@@ -1 +1,27 @@
-go.sum
+# Created by https://www.toptal.com/developers/gitignore/api/go
+# Edit at https://www.toptal.com/developers/gitignore?templates=go
```

Comment:
I think you should add more options to generate `.gitignore` file. [Here](https://www.toptal.com/developers/gitignore/api/go,visualstudiocode,windows,macos,linux,goland+all) is what I usually use.